cmd/swapcli/main.go

```diff
@@ -545,7 +546,7 @@ func runMake(ctx *cli.Context) error {
 		printOfferSummary(resp)
 
 		for stage := range statusCh {
-			fmt.Printf("> Stage updated: %s\n", stage)
+			fmt.Printf("%s > Stage updated: %s\n", time.Now().Format("2006-01-02T15:04:05"), stage)
```

Comment:
We have a constant for that formatting option, `common.TimeFmtSecs`. Your call on whether to use it. My thought was that if we ever wanted to update how time is formatted in one place, the constant might help. It's not a guarantee that we always want to use the same format everywhere though.

Reply:
I had some extra time, so I just replaced the format strings without our constant. I didn't realize it initially, but the format you had was slightly different than the constant:

Our constant:
```
2023-02-18-16:41:17
```
Your initial version:
```
2023-02-18T16:41:17
```
If you like the `T` better than the `-`, let's update the constant so we use the same format everywhere.